community/examples/AMD/hpc-cluster-amd-slurmv5.yaml

```diff
@@ -0,0 +1,125 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: hpc-cluster-amd
+
+vars:
+  project_id: ## Set GCP Project ID Here ##
+  deployment_name: amd-v5
+  region: us-east4
+  zone: us-east4-c
+
+deployment_groups:
+- group: primary
+  modules:
+  - source: modules/network/vpc
+    id: network1
+  - source: modules/file-system/filestore
+    id: homefs
+    use: [network1]
+    settings:
+      local_mount: /home
+  - source: modules/file-system/filestore
+    id: swfs
+    use: [network1]
+    settings:
+      local_mount: /sw
+  - source: community/modules/scripts/spack-install
+    id: spack
+    settings:
+      install_dir: /sw/spack
+      spack_url: https://github.com/spack/spack
+      spack_ref: v0.18.1
+      configs:
+      - type: file
+        scope: site
+        content: |
+          packages:
+            slurm:
+              externals:
+                - spec: slurm@22-05-2
+                  prefix: /usr/local
+              buildable: False
+      log_file: /var/log/spack.log
+      compilers: []
+      packages: []
+  - source: modules/scripts/startup-script
+    id: spack-startup
+    settings:
+      runners:
+      - type: shell
+        source: modules/startup-script/examples/install_ansible.sh
+        destination: install_ansible.sh
+      - type: ansible-local
+        source: modules/spack-install/scripts/install_spack_deps.yml
+        destination: install_spack_deps.yml
+      - type: shell
+        content: $(spack.startup_script)
+        destination: install_spack.sh
+      # the following installation of AOCC may be automated in the future
+      # with a clear direction to the user to read the EULA at
+      # https://developer.amd.com/aocc-compiler-eula/
+      - type: data
+        destination: /var/tmp/install_aocc.sh
+        content: |
+          #!/bin/bash
+          spack install aocc@3.2.0 +license-agreed
+          spack load aocc@3.2.0
+          spack compiler find --scope site
+          spack -d install -v openmpi@4.1.3 %aocc@3.2.0 +legacylaunchers +pmi schedulers=slurm
+  - source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    id: low_cost_partition
+    use:
+    - network1
+    - homefs
+    - swfs
+    settings:
+      partition_name: lowcost
+      machine_type: c2d-standard-4
+      node_count_dynamic_max: 10
+      enable_placement: false
+      is_default: false
```

Comment:
`is_default` [defaults](https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/compute/schedmd-slurm-gcp-v5-partition#input_is_default) to false. Unless included for clarity, this line can be removed.